examples/ShoppingAds/AddPerformanceMaxRetailCampaign.cs

```diff
@@ -1012,6 +1014,13 @@ public string Next()
 
                 // Unset for the root listing group filter node.
                 // ParentListingGroupFilter = "<PARENT FILTER NAME>"
+
+                // The Subdivision type means this node has children.
```

Comment:
The comment mentions this node has children, but I don't see children being added. Will this mutate succeed? With [ListingGroupInfo](https://developers.google.com/google-ads/api/reference/rpc/v10/ListingGroupInfo), the API will reject an attempt to add a `SUBDIVISION` listing group that doesn't have any children.